sw/device/lib/testing/clkmgr_testutils.h

```diff
@@ -0,0 +1,41 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_CLKMGR_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_CLKMGR_TESTUTILS_H_
+
+#include "sw/device/lib/dif/dif_base.h"
+#include "sw/device/lib/dif/dif_clkmgr.h"
+
+/**
+ * Returns the transactional block's clock status.
+ *
+ * @param clkmgr A clkmgr handle.
+ * @param params Hardware instance parameters.
```

Comment:
These are a weird thing, and I don't really understand what they are for, but this comment and the function declaration below have parameter or some abbreviation 5 times in 2 different contexts and are making it worse.

Reply:
Removing params, as suggested above, will help.

Reply:
Same comment as above - this is documented here:
https://docs.opentitan.org/sw/device/lib/dif/README/index.html#base-types

The DIF invoked by this function needs a `dif_clkmgr_params_t` arg. I am just wiring it up to this function. The goal of this PR is to not fix the DIFs, but to introduce some reusable test code. I think that should be a separate PR (I will recommend you to prepare a PR making this change if you think the current `clkmgr` DIFs don't seem right to you).